### PR TITLE
Don't use offensive wording in comments

### PR DESCRIPTION
**Pull request by fritsch**
In recent discussions it came up that we should use a more polite way of speaking. Here is take 1.

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
Sorry but a PR to just rephrase something in a non "offensive"  way is kind of wasting time without actually fixing the root cause. Sadly the time now spend on this PR has spend some valuable time and has also used up internet/server/laptop time which use lots of energy which does not fit within our environmental friendly ethos.

**Comment by notspiff** [comment]
Martijn, be considerate of other's feelings. You should not tell people
they are wasting their time.

**Comment by hudokkow** [comment]
Or energy. Or electricity. Or ...

File: xbmc/guilib/guiinfo/SystemGUIInfo.cpp [hunk 1]
```diff
@@ -425,7 +425,8 @@ bool CSystemGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
       value = false;
       return true;
     case SYSTEM_ETHERNET_LINK_ACTIVE:
-      // wtf: not implementated - always returns true?!
+      // not implementated - always returning true is bad practice though
```
Comment (hudokkow) [inline xbmc/guilib/guiinfo/SystemGUIInfo.cpp]: Let's review the review! s/not implementated/not implemented/

I'm cooking at the same time.

File: xbmc/platform/posix/filesystem/SMBFile.cpp [hunk 2]
```diff
@@ -39,9 +39,9 @@ void xb_smbc_auth(const char *srv, const char *shr, char *wg, int wglen,
   return ;
 }
 
-// WTF is this ?, we get the original server cache only
-// to set the server cache to this function which call the
-// original one anyway. Seems quite silly.
+// We get the original server cache only
+// to set the server cache to this function, which call the
```
Comment (hudokkow) [inline xbmc/platform/posix/filesystem/SMBFile.cpp]: s/which call the/which calls the/

**Review by hudokkow — CHANGES_REQUESTED** [review]

File: xbmc/guilib/guiinfo/SystemGUIInfo.cpp (hunk 1 above)
Comment (fritsch) [reply xbmc/guilib/guiinfo/SystemGUIInfo.cpp]: Thanks very much for your valuable comment. I wanted to honor the original author, cause he might have a specialy feeling when using "implemented" in a wrong form.

Do you think it is okay to change it?

**Comment by Jalle19** [comment]
This should hopefully fix the crashes I've been experiencing recently

File: xbmc/guilib/guiinfo/SystemGUIInfo.cpp (hunk 1 above)
Comment (ksooo) [reply xbmc/guilib/guiinfo/SystemGUIInfo.cpp]: I'm the original author of that bad phrase and I do really sorry for using hate speech and even wrong spelling. Yes, both the hate speech and the typo should be fixed asap. Thanks for pointing this out.

**Comment by hudokkow** [comment]
Now I feel offended. My handle is `hudo`, not `hudoo`.

Please correct ASAP.

**Comment by fritsch** [comment]
Really sorry - the many ooos of @ksooo confused me.

**Review by hudokkow — APPROVED** [review]

**Comment by hudokkow** [comment]
Nicely done! Thank you for your contributions to Kodi. Much appreciated.

I feel we should go further in future PRs and push for correctly capitalized sentences in comments. Thoughts?

**Comment by hudokkow** [comment]
yeah... Sounds wrong to me. Maybe `derriereumptions` but I'm not a native speaker.

**Comment by garbear** [comment]
Good job turning useless comments into legitimately useful ones.

**Comment by HeresJonny** [comment]
All this is kinda cute but has nothing to do with hate speech. Hate speech is directed at people or groups of people. What you're fixing here is a bit of crude language... which noone should care about in comments.

File: xbmc/music/tags/TagLoaderTagLib.cpp [hunk 3]
```diff
@@ -423,7 +423,7 @@ bool CTagLoaderTagLib::ParseTag(ID3v2::Tag *id3v2, EmbeddedArt *art, MUSIC_INFO:
 
         // @xbmc.org ratings trump others (of course)
         if      (popFrame->email() == "ratings@xbmc.org")
-          tag.SetUserrating(popFrame->rating() / 51); //! @todo wtf? Why 51 find some explanation, somewhere...
+          tag.SetUserrating(popFrame->rating() / 51); // Scale is: 1 -> 51, 1.5 -> 76, 2 -> 102, ...
```
Comment (Jalle19) [inline xbmc/music/tags/TagLoaderTagLib.cpp]: Maybe the `@todo` should be left in so it's easier to find?

Comment (fritsch) [reply xbmc/music/tags/TagLoaderTagLib.cpp]: I fixed the todo and explained why one needs to divide by 51.

**Comment by fritsch** [comment]
@HeresJonny Btw. a code comment is directed from one developer to a another developer or even an entire group of people reading this source code. Comments are always towards So also comments would fall into your classification. I changed the PR and the commits never the less.

File: xbmc/music/tags/TagLoaderTagLib.cpp (hunk 3 above)
Comment (Jalle19) [reply xbmc/music/tags/TagLoaderTagLib.cpp]: Okay

**Review by Memphiz — APPROVED** [review]
I don‘t give a ... care :)

**Comment by da-anda** [comment]
should we rename "blacklist" to "non-whitelist", since it could offend a certain community?

**Comment by FernetMenta** [comment]
> should we rename "blacklist" to "non-whitelist", since it could offend a certain community?

even worse, then it could be interpreted as offending by all but a certain community

**Comment by jimfcarroll** [comment]
This PR offends me

**Comment by fritsch** [comment]
@jimfcarroll Do you have a suggestion how we can change "the wording" so that it would be good to go with you and also have a probability > 99.9999 % that it won't offend others?

**Comment by fritsch** [comment]
Thanks for everyone helping out in this social study. I think we are well prepared for Code of Conducts as there is a lot of humor, sarcasm and the ability to laugh about ourselves left :-)

Don't be evil - don't assume evil.

**Comment by MartijnKaijser** [comment]
Well at least can you update the comments about the rating stuff you figured out? Thx

**Comment by hudokkow** [comment]
Did I do something wrong? I approved this PR and now I feel ostracized... :cry: 

You guys don't trust my judgment... That's it! You don't like me! Shame on you!

:rose:

**Pull request closed without merging**